moto/memorydb/models.py

```diff
@@ -67,14 +67,17 @@ def __init__(
         self.maintenance_window = maintenance_window or "wed:08:00-wed:09:00"
         self.port = port or 6379  # Default is set to 6379
         self.sns_topic_arn = sns_topic_arn
-        self.tls_enabled = tls_enabled or True
+        self.tls_enabled = True
+        self.acl_name = acl_name
```

Comment:
Hi @archinksagar, can I suggest a change that's closer to what we had before?

```suggestion
        self.tls_enabled = tls_enabled if tls_enabled is not None else True
        self.acl_name = "open-access" if tls_enabled is False else acl_name
```
Bit of a nitpick, but this is much more readable IMO - makes it more clear what's happening

Same for the `auto_minor_version_upgrad` below.

Reply:
@bblommers , Thanks for the suggestion, i modified the code accordingly, please take a look